lms/static/js/verify_student/views/step_view.js

```diff
@@ -101,4 +103,4 @@
 
     });
 
- })( jQuery, _, _.str, Backbone, gettext );
+ })( jQuery, _, Backbone, gettext );
```

Comment:
Note to reviewers: when this code loads while running tests, _.str has not yet been defined. It is though by the time the code executes, so I just changed the code to get it directly from _.